docker-compose.dev.override.yml

```diff
@@ -1,23 +1,36 @@
 version: '3.4'
 
+x-airflow-env:
+  &airflow-env
+    - CIVICRM_API_KEY_FILE_PATH=/usr/local/airflow/secret/civi_api_key
+    - CIVICRM_SITE_KEY_FILE_PATH=/usr/local/airflow/secret/civi_site_key
+    - PATH_TO_TOGGL_API_TOKEN_FILE=/usr/local/airflow/toggl.txt
+
 
 x-airflow-volumes:
   &airflow-volumes
     - ~/.config/gcloud/application_default_credentials.json:/tmp/credentials.json
+    - ./civi_site_key:/usr/local/airflow/secret/civi_site_key
```

Comment:
minor: we probably need some developer documentation relating to those keys. The gcloud and AWS credentials are pretty standard but these are not.
I suspect if they don't exist, then Docker will create directories with those names which can be confusing. Maybe if they live locally, it could be easier to mount a `.secrets` directory with those three files in it.
What do you think?

Reply:
moved them all to a directory